src/ui/tsconfig.json

```diff
@@ -1,12 +0,0 @@
-{
-	"compilerOptions": {
-		"module": "none",
-		"composite": true,
-		"target": "es5",
-		"lib": [
-			"dom"
-		],
-		"moduleResolution": "node",
-	},
-	"files": [ "login.ts", "admin.ts" ]
-}

```

Comment:
Just a heads up: compiling the client-side JS with a different `tsconfig.json` was intentional. The compile target is the near-universal ES5 (so TypeScript will down-compile promises for example to browsers that don't support it), the `dom` library is included (which the Node.js code cannot use), and the CommonJS module system is disabled (not supported in browsers).

Reply:
Ok thanks for letting me know, I fixed it by adding in gulp and babel, and this way everything will compile to a dist directory instead of having the compilations in the src directory and having the .js files mess up the structure. I also added in ts-node and nodemon for working in dev.

Reply:
I also moved the bugsnag build reporting to the gulp build process